### ISSUE TITLE
Invisible mesh caused from importing raw issue

### ISSUE THREAD

**Issue by ZahidRiefqi** [SeriousCache/UABE#572]
I tried importing my own model by exporting raw to the base game but the thing is after I tried to look in the game it didnt show anything instead there's nothing at all, it's just invisible mesh for some reason. For clarification I use 3.0 build and also use the same unity game engine version as the base game itself.

**Comment by Pirangunter21** [comment]
Turns out the reason why mesh models are invisible is because the mesh are skinned mesh with certain avatar, and to replace this you need to decompile the game data to unity asset and re-convert the file (was prefab) to .fbx  in unity game engine with the same version as the game. That way you can replace skinned mesh properly. 

tl;dr you need to use asset ripper & asset extractor to convert the file to .asset package then convert desired prefab parented to skinned mesh renderer to .fbx then you can replace the model with custom one
